docs/UserGuide.md

```diff
@@ -53,8 +53,13 @@ Interface (GUI). If you can type fast, Duke The Market can help you organise you
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* If a parameter is specified multiple times, only the last occurrence of the parameter will be considered regardless of the validity of previous parameter inputs.<br>
+  e.g. if you specify `p/12341234 p/56785678`, even though both are valid phone numbers,
+       only `p/56785678`, will be taken by the command as its parameter input
+  e.g. if you specify `d/A/B/C d/01/01/2022`, even though the first parameter input `d/A/B/C` is invalid,
+       only `d/01/01/2022`, which is a valid parameter input, will be taken by the command as its parameter input
+  e.g. if you specify `d/01/01/2022 d/A/B/C`, even though the first parameter input `d/01/01/2022` is valid, 
+       only `d/A/B/C`, which is an invalid parameter input, is considered and thus outputs an error message.
```

Comment:
If we're not specifying the output for the first 2 examples (i.e whether success or error) would be better to leave it out here as well
```suggestion
       only `d/A/B/C`, which is an invalid parameter input, is considered.
```